web/src/connectors/task-annotator-connector/task-annotator-utils.ts

```diff
@@ -58,7 +58,7 @@ export const getCategoryDataAttrs = (
 
     if (!categoryDataAttrsCache.has(annotationLabel)) {
         const foundCategoryDataAttrs = categories?.find(
-            (el) => el.name.toString() === annotationLabel
+            (el) => el.id.toString() === annotationLabel
```

Comment:
1. Let's rename annotationLabel to annotationCategory
2. Also we need to call getCategoryDataAttrs with category, not label
![image](https://user-images.githubusercontent.com/69203629/210531291-ea5b4a93-79d3-4e0a-9ad1-e2cb5a9ded9e.png)